src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -8,11 +9,18 @@ import {
   MODULE_PATHS,
 } from '../../utils'
 
-export const messageId = 'preferConcatLatestFrom'
-export const messageIdSuggest = 'preferConcatLatestFromSuggest'
-export type MessageIds = typeof messageId | typeof messageIdSuggest
+export const preferConcatLatestFrom = 'preferConcatLatestFrom'
+export const preferConcatLatestFromSuggest = 'preferConcatLatestFromSuggest'
+export type MessageIds =
+  | typeof preferConcatLatestFrom
+  | typeof preferConcatLatestFromSuggest
 
-type Options = []
+type Options = [{ readonly strict: boolean }]
+const defaultOptions: Options[number] = { strict: true }
```

Comment:
Should the default be `true` or `false`?

Reply:
I think we should have `false` as default, we can always reevaluate later.
I also want to test this on a few projects 😅